README.md

```diff
@@ -13,7 +13,8 @@ For more information: https://hackmd.io/@Vid201/aa-bundler-rust
 ## Prerequisites
 
 1. Ethereum JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall) (currently implemented only in [Geth](https://github.com/ethereum/go-ethereum) and [Erigon](https://github.com/ledgerwatch/erigon)). For testing purposes, you can setup [private Geth node](https://github.com/krzkaczor/geth-private-node).  
-1. [solc](https://docs.soliditylang.org/en/v0.8.17/installing-solidity.html) >=0.8.12
+2. [solc](https://docs.soliditylang.org/en/v0.8.17/installing-solidity.html) >=0.8.12
+3. For testing purposes, we need [Anvil](https://github.com/foundry-rs/foundry/tree/master/anvil#anvil) which would provide enough functions of execution client like Geth. To install Anvil, you can setup with [foundryup](https://getfoundry.sh/)
```

Comment:
I would combine 1. and 3. Something like that:

1. Ethereum execution client JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall). For production, you can use [Geth](https://github.com/ethereum/go-ethereum) or [Erigon](https://github.com/ledgerwatch/erigon). For testing purposes, you can use [Anvil](https://github.com/foundry-rs/foundry/tree/master/anvil#anvil), which provides enough functionalities of execution clients like Geth. To install Anvil, use [foundryup](https://getfoundry.sh/)

Reply:
Cool. Love your version.